src/sentry/eventstore/reprocessing/redis.py

```diff
@@ -163,12 +166,11 @@ def start_reprocessing(
             ),
         )
 
-    def get_pending(self, group_id: int) -> tuple[int | None, int]:
+    def get_pending(self, group_id: int) -> tuple[str | None, int]:
         pending_key = _get_sync_counter_key(group_id)
         pending = self.redis.get(pending_key)
         ttl = self.redis.ttl(pending_key)
         return pending, ttl
 
-    def get_progress(self, group_id: int) -> dict[str, Any] | None:
-        info = self.redis.get(_get_info_reprocessed_key(group_id))
-        return info
+    def get_progress(self, group_id: int) -> str | None:
+        return self.redis.get(_get_info_reprocessed_key(group_id))
```

Comment:
This looks correct, seems like service wrapper abstractions led to types being lost which is why the incorrect return type wasn't found.

Reply:
Thank you for reviewing Mark. Actually, this was raising mypy errors.

I've updated this to call `json.loads()` here since the inherited class has `dict[str, Any]` signature. Let me know if the new change is correct.